### DISCUSSION TITLE
Help me please....How do I fix this error?

### DISCUSSION THREAD

**Discussion by WCatherineW** [openai/whisper#2466] [Q&A]
What is this anyway?
![error](https://github.com/user-attachments/assets/64b92a34-3e1e-4e71-9f5c-487b8b07ec62)

**Comment by glangford** [comment]
There are two warnings (and they are only warnings so you could just ignore them).

The first one is a FutureWarning from your version of pytorch, it is discussed here:
* https://github.com/openai/whisper/discussions/2462#discussioncomment-11430277

There is a workaround to make this warning go away in that discussion, or you can install an older version of pytorch. 

The second warning (FP16 is not supported on CPU) is discussed here:
* https://github.com/openai/whisper/discussions/1194
